### PR TITLE
🎨 Made the requested changes

### PR DESCRIPTION
**Pull request by Musa-Mabasa**

### CODE REVIEW

File: src/features/mangaViewer/MangaViewer.tsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ const MangaViewer = () => {
           }`}
           alt=""
         />
-        <div className="h-[80%] w-full absolute z-40 bottom-0 bg-gradient-to-t from-primary-color from-20% via-gradient-color via-85%   "></div>
+        <div className="h-4/5 w-full absolute z-40 bottom-0 bg-gradient-to-t from-primary-color from-20% via-gradient-color via-85%   "></div>
```
Comment (yanevdw) [inline src/features/mangaViewer/MangaViewer.tsx]: Just check the spacing behind here.

File: src/features/mangaViewer/components/ChaptersTab.tsx [hunk 2]
```diff
@@ -12,13 +12,13 @@ const ChaptersTab = ({ mangaDetails }: { mangaDetails: MangaDetails }) => {
...
       <div className="flex justify-center items-center grow">
-        <span className=" loading loading-dots loading-lg text-secondary-color"></span>
+        <span className="loading loading-dots loading-lg text-secondary-color"></span>
       </div>
     );
   }
 
   if(isError){
-    return <div className="h-full flex items-center justify-center"> NO manga</div>
+    return <h1 className="h-full flex items-center justify-center"> No manga</h1>
```
Comment (yanevdw) [inline src/features/mangaViewer/components/ChaptersTab.tsx]: Is the spacing in front of "no" intentional? If so, just disregard this comment.

File: src/features/mangaViewer/components/SimilarTab.tsx [hunk 3]
```diff
@@ -10,7 +10,7 @@ const SimilarTab = ({
 }) => {
   if (recommendations?.length === 0) {
     return (
-      <div className="h-full flex items-center justify-center"> NO manga</div>
+      <h1 className="h-full flex items-center justify-center"> NO manga</h1>
```
Comment (yanevdw) [inline src/features/mangaViewer/components/SimilarTab.tsx:13]: For consistency purposes, maybe do "No" like above. Also, just check the spacing in front of "no" as mentioned above.

**Review by yanevdw — APPROVED** [review]
Requested changes have been made. Just check my comments related to some redundant spaces.

File: src/features/mangaViewer/MangaViewer.tsx (hunk 1 above)
Comment (Musa-Mabasa) [reply src/features/mangaViewer/MangaViewer.tsx]: ohh at the end. I kept wondering where it was. Thank you

File: src/features/mangaViewer/components/ChaptersTab.tsx (hunk 2 above)
Comment (Musa-Mabasa) [reply src/features/mangaViewer/components/ChaptersTab.tsx]: Ahh, I see it now, thank you

**Review by Meggie-H — APPROVED** [review]
Cool!, well done

**Pull request merged**